pkg/placement/membership.go

```diff
@@ -0,0 +1,146 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package placement
+
+import (
+	"time"
+
+	"github.com/dapr/dapr/pkg/placement/raft"
+	v1pb "github.com/dapr/dapr/pkg/proto/placement/v1"
+)
+
+const (
+	commandRetryInterval = 20 * time.Millisecond
+	commandRetryMaxCount = 3
+)
+
+// MembershipChangeWorker is the worker to change the state of membership
+// and update the consistent hashing tables for actors.
+func (p *Service) MembershipChangeWorker() {
+	faultyHostDetectTimer := time.NewTicker(faultyHostDetectInterval)
+	disseminateTimer := time.NewTicker(disseminateTimerInterval)
+
+	p.hostUpdateCount = 0
+
+	for {
+		if !p.raftNode.IsLeader() {
+			// TODO: use leader election channel instead of sleep
+			// <- p.raftNode.leaderCh
+			time.Sleep(100 * time.Millisecond)
```

Comment:
Can this be a const?

Reply:
I will get rid of this code in the next PR.